### ISSUE TITLE
[Documentation] Images in developer docs lack meaningful alt text

### ISSUE THREAD

**Issue by akhenry** [nasa/openmct#919]
The images in our documentation have alt text, but it's as good as useless. For integration with the web site, we will need to be compliant with https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-section-508-standards/guide-to-the-section-508-standards/web-based-intranet-and-internet-information-and-applications-1194-22
